domains/elarifamine.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "AmineArif1",
+        "email": "elarifamine1@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "aminearif1.github.io"
```

Comment:
404?

Reply:
Hey William , 
Sorry for that , i updated the link!
thanks.